tests/components/hue/conftest.py

```diff
@@ -1,11 +1,95 @@
 """Test helpers for Hue."""
-from unittest.mock import patch
+from collections import deque
+from unittest.mock import Mock, patch
 
+from aiohue.groups import Groups
+from aiohue.lights import Lights
+from aiohue.sensors import Sensors
 import pytest
 
+from homeassistant import config_entries
+from homeassistant.components import hue
+from homeassistant.components.hue import sensor_base as hue_sensor_base
+
 
 @pytest.fixture(autouse=True)
 def no_request_delay():
     """Make the request refresh delay 0 for instant tests."""
     with patch("homeassistant.components.hue.light.REQUEST_REFRESH_DELAY", 0):
         yield
+
+
+def create_mock_bridge(hass):
+    """Create a mock Hue bridge."""
+    bridge = Mock(
+        hass=hass,
+        available=True,
+        authorized=True,
+        allow_unreachable=False,
+        allow_groups=False,
+        api=Mock(),
+        reset_jobs=[],
+        spec=hue.HueBridge,
```

Comment:
We should not mock our own code. We should patch and mock the library code. That will make the tests more robust.